src/annotator/highlighter.ts

```diff
@@ -12,6 +12,57 @@ type HighlightProps = {
 
 export type HighlightElement = HTMLElement & HighlightProps;
 
+/**
+ * Return the closest generation of HighlightElements to `element`.
+ *
+ * If `element` is itself a HighlightElement, return immediate children that
+ * are also HighlightElements.
+ *
+ * Otherwise, return all HighlightElements that have no parent HighlightElement,
+ * i.e. the outermost highlights within `element`.
+ */
+export function getHighlights(element: Element) {
+  let highlights;
+  if (element.tagName === 'HYPOTHESIS-HIGHLIGHT') {
```

Comment:
`Element.tagName` can return a lower-case value in XHTML documents. See http://localhost:3000/document/xhtml-test and another instance of this issue: https://github.com/KoryNunn/scroll-into-view/issues/101.

Reply:
I think `Element.localName` will always return lower-case, but you'd need to verify. See https://developer.mozilla.org/en-US/docs/Web/API/Element/localName.